infra/datalake/datalake_stack.py

```diff
@@ -0,0 +1,95 @@
+            bucket_name="{}-{}".format(
+                self.node.try_get_context("data-lake-storage-bucket-name"), ENV
+            ),
+            access_control=aws_s3.BucketAccessControl.PRIVATE,
+            block_public_access=aws_s3.BlockPublicAccess.BLOCK_ALL,
+            versioned=True,
+            removal_policy=REMOVAL_POLICY,
+        )
+        Tags.of(storage_bucket).add("ApplicationLayer", "storage")
+
+        ############################################################################################
+        # ### APPLICATION DB #######################################################################
+        ############################################################################################
+        app_db_datasets = aws_dynamodb.Table(
+            self,
+            "data-lake-application-db",
+            table_name="datasets",
+            partition_key=aws_dynamodb.Attribute(name="pk", type=aws_dynamodb.AttributeType.STRING),
+            sort_key=aws_dynamodb.Attribute(name="sk", type=aws_dynamodb.AttributeType.STRING),
+            point_in_time_recovery=True,
+            removal_policy=REMOVAL_POLICY,
+        )
+
+        app_db_datasets.add_global_secondary_index(
+            index_name="datasets_title",
+            partition_key=aws_dynamodb.Attribute(name="sk", type=aws_dynamodb.AttributeType.STRING),
+            sort_key=aws_dynamodb.Attribute(name="title", type=aws_dynamodb.AttributeType.STRING),
+        )
+        app_db_datasets.add_global_secondary_index(
+            index_name="datasets_owning_group",
+            partition_key=aws_dynamodb.Attribute(name="sk", type=aws_dynamodb.AttributeType.STRING),
+            sort_key=aws_dynamodb.Attribute(
+                name="owning_group", type=aws_dynamodb.AttributeType.STRING
+            ),
+        )
+
+        Tags.of(app_db_datasets).add("ApplicationLayer", "application-db")
+
+        ############################################################################################
+        # ### API ENDPOINTS ########################################################################
+        ############################################################################################
+
+        endpoints = ("datasets",)
+
+        for endpoint in endpoints:
+            endpoint_function = aws_lambda.Function(
+                self,
+                f"{endpoint}-endpoint-function",
+                function_name=f"{endpoint}-endpoint",
+                handler=f"endpoints.{endpoint}.entrypoint.lambda_handler",
+                runtime=aws_lambda.Runtime.PYTHON_3_6,
+                code=aws_lambda.Code.from_asset(
+                    path="..",
+                    bundling=core.BundlingOptions(
+                        # pylint:disable=no-member
+                        image=aws_lambda.Runtime.PYTHON_3_6.bundling_docker_image,
+                        command=["backend/endpoints/bundle.bash", f"{endpoint}"],
```

Comment:
`f"{endpoint}"` can be just `endpoint`

Reply:
ah yes, thanks.

Reply:
Fixed in c30b2b91eb9adc7bf5a4ef2a8cfc99d59bd5cc94